worlds/terraria/Checks.py

```diff
@@ -0,0 +1,362 @@
+from BaseClasses import Location, Item
+
+items = [
```

Comment:
Would recommend renaming this to `prehardmode_items` for consistency of naming schema. Same applies to the various prehardmode location lists.